app/gui/e2e/project-view/nodeComments.spec.ts

```diff
@@ -0,0 +1,75 @@
+import test from 'playwright/test'
+import * as actions from './actions'
+import { expect } from './customExpect'
+import { CONTROL_KEY } from './keyboard'
+import * as locate from './locate'
+
+test('Edit comment by click', async ({ page }) => {
+  await actions.goToGraph(page)
+  const nodeComment = locate.nodeComment(locate.graphNodeByBinding(page, 'final'))
+  await expect(nodeComment).toHaveText('This node can be entered')
+
+  await nodeComment.click()
+  await page.keyboard.press(`${CONTROL_KEY}+A`)
```

Comment:
Just checking, as I don't know exactly how it works: if the click changes some HTML elements (like, the editable input appears only after clicking), we should add an expect here, otherwise Ctrl+A may be applied before input appears resulting in flaky test.